src/xtgeo/grid3d/grid_properties.py

```diff
@@ -8,7 +8,6 @@
 import deprecation
 import numpy as np
 import pandas as pd
-
```

Comment:
This PR should probably not contain any changes to this file.

Reply:
A.a